signature.py

```diff
@@ -397,6 +428,9 @@ def get_documents(self):
         return self.__class__.get_content_from_response(conf['provider'],
             response)
 
+    def get_forced_by(self, name):
+        return self.write_uid.id if self.write_uid else self.create_uid.id
```

Comment:
Ca devrait être un vrai champ, en soit il peut y avoir une modif post forçage

Reply:
Pour l'instant. On risque de casser ça si on rajoute un truc plus tard, sans s'en rendre compte, et en cassant une trace d'audit